packages/pages/src/App.js

```diff
@@ -4,46 +4,71 @@ import { format, parse } from "json-rpc-protocol";
 
 const App = ({ effects, state }) => (
   <div>
-    <div name="listEntries">
-      <button type="button" onClick={effects.refreshEntries}>
-        Refresh
-      </button>
-      <ul>
-        {state.entries.map(entry => (
-          <li key={entry.id}>
-            {entry.name} {entry.content}
-            <button
-              type="button"
-              value={entry.id}
-              onClick={effects.deleteEntry}
-            >
-              Delete
+    {state.isEditting ? (
+      <form onSubmit={effects.submitUpdates}>
```

Comment:
Good idea to use a form, it's more semantic and allows the user to submit simply by using the *Enter* key. :+1: